apps/meteor/client/hooks/useAnonymousUser.ts

```diff
@@ -0,0 +1,9 @@
+import { useMethod } from '@rocket.chat/ui-contexts';
+import { useMutation } from '@tanstack/react-query';
+
+export const useAnonymousUser = () => {
```

Comment:
The hook's name is not very informative. Why not `useRegisterAnonymousUserMutation`? It adds a verb and it declares the return type is not a simple callback.